### ISSUE TITLE
Tooltips Stop Working When Using iFrameResize with Embedded Metabase Dashboards

### ISSUE THREAD

**Issue by joaquinjsb** [metabase/metabase#49219]
### Describe the bug

Hello,
when embedding a static dashboard, the popup that appears on hover is not visible when iframeResizer is being used.

it's working on the embed url when navigating to the url of the iframe, or when the hosted iframe isn't using iframeResizer, also works on the dashboard/question on the mb instance.

![Image](https://github.com/user-attachments/assets/2ecff1ae-28f6-4cab-a2d4-13c283313053)
as you can see, the hover is in place but the pop-up doesn't appear.

### To Reproduce

1. Go to a dashboard
2. create a static embed
3. create a simple html page with an iframe with the metabase url and add the iframeresizer (<script src="http://metabaseinstance/app/iframeResizer.js"></script>
4. open the html page
5. hover on a chart.
6. no pop-up appears.

### Expected behavior

![Image](https://github.com/user-attachments/assets/2b051899-9121-44c3-998d-6dfcad14dedd)

### Logs

```

 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'nonce-D6qzYMYshE'   https://accounts.google.com". Either the 'unsafe-inline' keyword, a hash ('sha256-bgUp5tqDCSC1xokY+fdYcHMM/Fap5RX+TCFkA0sXuRk='), or a nonce ('nonce-...') is required to enable inline execution.

e @ vendor.66cc5cf07b44f626.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'nonce-HzzqMf4edF'   https://accounts.google.com". Either the 'unsafe-inline' keyword, a hash ('sha256-bgUp5tqDCSC1xokY+fdYcHMM/Fap5RX+TCFkA0sXuRk='), or a nonce ('nonce-...') is required to enable inline execution.

e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
e @ vendor.66cc5cf07b44f626.js:1
aJ @ vendor.66cc5cf07b44f626.js:1
aZ @ vendor.66cc5cf07b44f626.js:1
aK @ vendor.66cc5cf07b44f626.js:1
aZ @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
sj @ vendor.66cc5cf07b44f626.js:1
sS @ vendor.66cc5cf07b44f626.js:1
w @ vendor.66cc5cf07b44f626.js:1
E @ vendor.66cc5cf07b44f626.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
nx @ app-embed.c17169454236fce9.js:1
nv @ app-embed.c17169454236fce9.js:1
nw @ app-embed.c17169454236fce9.js:1
ly.O.g @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
t @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
r @ app-embed.c17169454236fce9.js:1
a @ app-embed.c17169454236fce9.js:1
p.kE @ app-embed.c17169454236fce9.js:1
q @ app-embed.c17169454236fce9.js:1
ly.g.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
l @ app-embed.c17169454236fce9.js:1
m.KZ.j @ app-embed.c17169454236fce9.js:1
m.KZ.g @ app-embed.c17169454236fce9.js:1
m.KZ.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
ly.g.h @ app-embed.c17169454236fce9.js:1
u @ app-embed.c17169454236fce9.js:1
er.M1 @ app-embed.c17169454236fce9.js:1
N.j @ app-embed.c17169454236fce9.js:1
e @ app-embed.c17169454236fce9.js:1
S @ app-embed.c17169454236fce9.js:1
query @ app-embed.c17169454236fce9.js:1
databaseId @ app-embed.c17169454236fce9.js:1
database @ app-embed.c17169454236fce9.js:1
m @ app-embed.c17169454236fce9.js:496
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
n.<computed> @ vendor.66cc5cf07b44f626.js:11
(anonymous) @ app-embed.c17169454236fce9.js:213
aj @ vendor.66cc5cf07b44f626.js:1
sH @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
sj @ vendor.66cc5cf07b44f626.js:1
sR @ vendor.66cc5cf07b44f626.js:1
r8 @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
nx @ app-embed.c17169454236fce9.js:1
nv @ app-embed.c17169454236fce9.js:1
nw @ app-embed.c17169454236fce9.js:1
ly.O.g @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
t @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
r @ app-embed.c17169454236fce9.js:1
a @ app-embed.c17169454236fce9.js:1
p.kE @ app-embed.c17169454236fce9.js:1
q @ app-embed.c17169454236fce9.js:1
ly.g.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
l @ app-embed.c17169454236fce9.js:1
m.KZ.j @ app-embed.c17169454236fce9.js:1
m.KZ.g @ app-embed.c17169454236fce9.js:1
m.KZ.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
ly.g.g @ app-embed.c17169454236fce9.js:1
I.Y2 @ app-embed.c17169454236fce9.js:1
nv @ app-embed.c17169454236fce9.js:1
nw @ app-embed.c17169454236fce9.js:1
ly.O.g @ app-embed.c17169454236fce9.js:1
n @ app-embed.c17169454236fce9.js:1
u @ app-embed.c17169454236fce9.js:1
er.M1 @ app-embed.c17169454236fce9.js:1
N.j @ app-embed.c17169454236fce9.js:1
e @ app-embed.c17169454236fce9.js:1
S @ app-embed.c17169454236fce9.js:1
query @ app-embed.c17169454236fce9.js:1
databaseId @ app-embed.c17169454236fce9.js:1
database @ app-embed.c17169454236fce9.js:1
m @ app-embed.c17169454236fce9.js:496
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
n.<computed> @ vendor.66cc5cf07b44f626.js:11
(anonymous) @ app-embed.c17169454236fce9.js:213
aj @ vendor.66cc5cf07b44f626.js:1
sH @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
sj @ vendor.66cc5cf07b44f626.js:1
sR @ vendor.66cc5cf07b44f626.js:1
r8 @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
nx @ app-embed.c17169454236fce9.js:1
nv @ app-embed.c17169454236fce9.js:1
nw @ app-embed.c17169454236fce9.js:1
ly.O.g @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
t @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
r @ app-embed.c17169454236fce9.js:1
a @ app-embed.c17169454236fce9.js:1
p.kE @ app-embed.c17169454236fce9.js:1
q @ app-embed.c17169454236fce9.js:1
ly.g.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
l @ app-embed.c17169454236fce9.js:1
m.KZ.j @ app-embed.c17169454236fce9.js:1
m.KZ.g @ app-embed.c17169454236fce9.js:1
m.KZ.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
ly.g.h @ app-embed.c17169454236fce9.js:1
u @ app-embed.c17169454236fce9.js:1
er.M1 @ app-embed.c17169454236fce9.js:1
N.j @ app-embed.c17169454236fce9.js:1
e @ app-embed.c17169454236fce9.js:1
S @ app-embed.c17169454236fce9.js:1
query @ app-embed.c17169454236fce9.js:1
databaseId @ app-embed.c17169454236fce9.js:1
database @ app-embed.c17169454236fce9.js:1
m @ app-embed.c17169454236fce9.js:496
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
n.<computed> @ vendor.66cc5cf07b44f626.js:11
(anonymous) @ app-embed.c17169454236fce9.js:213
aj @ vendor.66cc5cf07b44f626.js:1
sH @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
sj @ vendor.66cc5cf07b44f626.js:1
sR @ vendor.66cc5cf07b44f626.js:1
r8 @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
 %c[%c%cmetabase.lib.normalize%c%c]%c %c"Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"%c color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#ffffff;background-color:#f5871f color:black;background-color:inherit color:#3e999f color:black
ny @ app-embed.c17169454236fce9.js:1
nx @ app-embed.c17169454236fce9.js:1
nv @ app-embed.c17169454236fce9.js:1
nw @ app-embed.c17169454236fce9.js:1
ly.O.g @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
t @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
r @ app-embed.c17169454236fce9.js:1
a @ app-embed.c17169454236fce9.js:1
p.kE @ app-embed.c17169454236fce9.js:1
q @ app-embed.c17169454236fce9.js:1
ly.g.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
l @ app-embed.c17169454236fce9.js:1
m.KZ.j @ app-embed.c17169454236fce9.js:1
m.KZ.g @ app-embed.c17169454236fce9.js:1
m.KZ.h @ app-embed.c17169454236fce9.js:1
(anonymous) @ app-embed.c17169454236fce9.js:1
ly.g.g @ app-embed.c17169454236fce9.js:1
I.Y2 @ app-embed.c17169454236fce9.js:1
nv @ app-embed.c17169454236fce9.js:1
nw @ app-embed.c17169454236fce9.js:1
ly.O.g @ app-embed.c17169454236fce9.js:1
n @ app-embed.c17169454236fce9.js:1
u @ app-embed.c17169454236fce9.js:1
er.M1 @ app-embed.c17169454236fce9.js:1
N.j @ app-embed.c17169454236fce9.js:1
e @ app-embed.c17169454236fce9.js:1
S @ app-embed.c17169454236fce9.js:1
query @ app-embed.c17169454236fce9.js:1
databaseId @ app-embed.c17169454236fce9.js:1
database @ app-embed.c17169454236fce9.js:1
m @ app-embed.c17169454236fce9.js:496
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
dispatch @ vendor.66cc5cf07b44f626.js:8
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ app-embed.c17169454236fce9.js:159
(anonymous) @ vendor.66cc5cf07b44f626.js:8
n.<computed> @ vendor.66cc5cf07b44f626.js:11
(anonymous) @ app-embed.c17169454236fce9.js:213
aj @ vendor.66cc5cf07b44f626.js:1
sH @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
sj @ vendor.66cc5cf07b44f626.js:1
sR @ vendor.66cc5cf07b44f626.js:1
r8 @ vendor.66cc5cf07b44f626.js:1
(anonymous) @ vendor.66cc5cf07b44f626.js:1
vendor.66cc5cf07b44f626.js:1 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'nonce-Mof5AYav0W'   https://accounts.google.com". Either the 'unsafe-inline' keyword, a hash ('sha256-bgUp5tqDCSC1xokY+fdYcHMM/Fap5RX+TCFkA0sXuRk='), or a nonce ('nonce-...') is required to enable inline execution.

e @ vendor.66cc5cf07b44f626.js:1
core.cljs:4009 [metabase.lib.normalize] "Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"
ny @ core.cljs:4009
core.cljs:4009 [metabase.lib.normalize] "Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"
ny @ core.cljs:4009
core.cljs:4009 [metabase.lib.normalize] "Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"
ny @ core.cljs:4009
core.cljs:4009 [metabase.lib.normalize] "Error normalizing pMBQL:\n{:value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n :schema #object[Object [object Object]],\n :explain\n {:schema #object[Object [object Object]],\n  :value {:lib/type :mbql/query, :stages [{:lib/type :mbql.stage/mbql}], :lib/metadata #object[b [object Object]]},\n  :errors\n  ({:path [0 0 :database],\n    :in [:database],\n    :schema #object[Object [object Object]],\n    :value nil,\n    :type :malli.core/missing-key}\n   {:path [0 0 :stages 0 1 0 0 :mbql.stage/mbql],\n    :in [:stages 0], \n    :schema #object[Object [object Object]], \n    :value {:lib/type :mbql.stage/mbql}})}}\n"
ny @ core.cljs:4009
html2canvas-pro.js:6063 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'nonce-Mof5AYav0W'   https://accounts.google.com". Either the 'unsafe-inline' keyword, a hash ('sha256-UP0QZg7irvSMvOBz9mH2PIIE28+57UiavRfeVea0l3g='), or a nonce ('nonce-...') is required to enable inline execution.

A.toIFrame @ html2canvas-pro.js:6063
Show 1 more frame
Show less
html2canvas-pro.js:6063 Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'nonce-Mof5AYav0W'   https://accounts.google.com". Either the 'unsafe-inline' keyword, a hash ('sha256-bgUp5tqDCSC1xokY+fdYcHMM/Fap5RX+TCFkA0sXuRk='), or a nonce ('nonce-...') is required to enable inline execution.

A.toIFrame @ html2canvas-pro.js:6063
Show 1 more frame
Show less



```

### Information about your Metabase installation

```json
{
  "browser-info": {
    "language": "it-IT",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "metabase-info": {
    "databases": [
      "oracle",
      "h2",
      "postgres"
    ],
    "run-mode": "prod",
    "plan-alias": "",
    "version": {
      "date": "2024-10-24",
      "tag": "v1.51.1.2",
      "hash": "4721f20"
    },
    "settings": {
      "report-timezone": null
    },
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "15rc2 (Debian 15~rc2-1.pgdg110+1)"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.7.3"
      }
    }
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.25+9",
    "java.vendor": "Eclipse Adoptium",
    "java.vendor.url": "https://adoptium.net/",
    "java.version": "11.0.25",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.25+9",
    "os.name": "Linux",
    "os.version": "5.14.0-427.18.1.el9_4.x86_64",
    "user.language": "en",
    "user.timezone": "GMT"
  }
}

```

### Severity

it prevents all users from getting an insight of the data they're viewing, for example it's not always possible to see which month a data point is refering to, etc.

**Comment by paoliniluis** [comment]
@joaquinjsb are you a paid customer? if so please contact us

**Comment by joaquinjsb** [comment]
yes I am, which form should I use?

**Comment by bshepherdson** [comment]
Triage note: I'm not sure if this is a regression from v50 -> v51 or not. I can't readily reproduce embedding issues.

**Comment by joaquinjsb** [comment]
I just did a test with v1.50.29 and v1.50.31.2, and it was working properly, after I upgraded to 1.51.x, the issue started.

**Comment by ixipixi** [comment]
Confirmed - can repro in 51.1 but not 50.30.

**Comment by WiNloSt** [comment]
Duplicate of https://github.com/metabase/metabase/issues/49537 and was fixed by https://github.com/metabase/metabase/pull/49964 which is included in [0.51.4](https://github.com/metabase/metabase/releases/tag/v0.51.4)
